### PR TITLE
Add star-check icon to Explore page

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Fixes https://github.com/red-hat-data-services/odh-dashboard/issues/22

image

### CODE REVIEW

**Comment by kywalker-rh** [comment]
The icon looks like its slightly higher in this implementation than on the Enabled page, is it possible to get the icon centered on the title?

Also, it is my understanding every Red Hat managed service is also Certified and Supported, so this icon should be applied to each card that also has the Red Hat managed label.

**Comment by jeff-phillips-18** [comment]
@kywalker-rh Updated. It seems `Pachyderm` should have been labelled as `Self managed`.

image

**Review by kywalker-rh — APPROVED** [review]
Looks great, thanks for making the updates!

File: frontend/src/components/OdhAppCard.tsx [hunk 1]
```diff
@@ -100,14 +100,16 @@ const OdhAppCard: React.FC<OdhAppCardProps> = ({ odhApp }) => {
...
-        <Tooltip content="Red Hat Certified and Supported">
-          <img
-            className={supportedImageClasses}
-            src="../images/CheckStar.svg"
-            alt="Red Hat Certified and Supported"
-          />
-        </Tooltip>
+        <span className="odh-card__title-supported">
+          {odhApp.spec.displayName}
+          <Tooltip content="Red Hat Certified and Supported">
```
Comment (jschuler) [inline frontend/src/components/OdhAppCard.tsx:105]: should this be sentence case?

File: frontend/src/components/OdhExploreCard.tsx [hunk 2]
```diff
@@ -42,14 +43,16 @@ const OdhExploreCard: React.FC<OdhExploreCardProps> = ({ odhApp, isSelected, onS
...
       <CardTitle>
-        {odhApp.spec.displayName}
-        <Tooltip content="Red Hat Certified and Supported">
-          <img
-            className={supportedImageClasses}
-            src="../images/CheckStar.svg"
-            alt="Red Hat Certified and Supported"
-          />
-        </Tooltip>
+        <span className="odh-card__title-supported">
```
Comment (jschuler) [inline frontend/src/components/OdhExploreCard.tsx:46]: could be a component to keep things DRY

**Review by jschuler — APPROVED** [review]
LGTM otherwise

File: frontend/src/components/OdhAppCard.tsx (hunk 1 above)
Comment (kywalker-rh) [reply frontend/src/components/OdhAppCard.tsx:105]: Good question! I'm checking with those that know better than me, and will reply with the answer.

Comment (kywalker-rh) [reply frontend/src/components/OdhAppCard.tsx:105]: Pretty sure we should do sentence case, but I'm waiting to hear back from OCM to see what they do.

**Comment by jeff-phillips-18** [comment]
@jschuler Updated per comments, PTAL

File: frontend/src/components/OdhAppCard.tsx (hunk 1 above)
Comment (kywalker-rh) [reply frontend/src/components/OdhAppCard.tsx]: Sentence case it is, thanks for spotting that @jschuler!

**Review by jschuler — APPROVED** [review]

**Pull request merged**